src/Iban/CzechAndSlovakIbanAdapter.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace Rikudou\Iban\Iban;
+
+use Rikudou\Iban\Helper\ToStringIbanTrait;
+use Rikudou\Iban\Validator\GenericIbanValidator;
+use Rikudou\Iban\Validator\ValidatorInterface;
+
+abstract class CzechAndSlovakIbanAdapter implements IbanInterface
```

Comment:
Can you mark this as internal?

```suggestion
/**
 * @internal
 */
abstract class CzechAndSlovakIbanAdapter implements IbanInterface
```